code/controllers/subsystem/weeds.dm

```diff
@@ -100,3 +102,18 @@ SUBSYSTEM_DEF(weeds)
 			swapped = TRUE
 			qdel(O)
 	new weed_to_spawn(T, node, swapped)
+
+/datum/controller/subsystem/weeds/proc/create_node(turf/T, obj/alien/weeds/node/node)
+	if(QDELETED(node) || (node.name != WEED) || !T.is_weedable() || iswallturf(T) || (locate(/obj/alien/weeds/node) in range(1, T)))
+		return
+	var/obj/alien/weeds/node/node_to_spawn = /obj/alien/weeds/node
+	for (var/obj/O in T)
+		if(O.density)
+			return
+		else if(istype(O, /obj/alien/weeds))
+			if(istype(O, /obj/alien/weeds/node))
+				return
+			var/obj/alien/weeds/weed = O
+			weed.swapped = TRUE
```

Comment:
Does this do anything if you are qdeleting it instantly ? I know it's my code but this looks useless